Add docs for log audit manager (#16927)
apache/druid: docs/configuration/index.md

|Property|Description|Default|
|--------|-----------|-------|

> |`druid.audit.manager.type`|Allows audited events to either be logged when set to `log` or persisted in metadata store when set to `sql`.|sql|

Reviewer: ```suggestion
|`druid.audit.manager.type`|Type of audit manager used for handling audited events. Audited events are logged when set to `log` or persisted in metadata store when set to `sql`.|sql|
```

> |`druid.audit.manager.logLevel`|Set the log level of audit events and can take values DEBUG, INFO, WARN. This gets activated only if type is `log`.|INFO|

Reviewer: ```suggestion
|`druid.audit.manager.logLevel`|Log level of audit events with possible values DEBUG, INFO, WARN. This property is used only when `druid.audit.manager.type` is set to `log`.|INFO|
```
